_gsocproposals/2020/proposal_XRootDKubernetes.md

```diff
@@ -32,9 +32,9 @@ A production-ready operator for XRootD, enabling at least **Basic Install** and
 [Go](https://golang.org), Shell scripting, [Kubernetes](https://kubernetes.io/), C++
 
 ## Mentors
-  * [Yvan Calas](mailto:yvan.calas@cc.in2p3.fr)
-  * [Andrew Hanushevsky](mailto:abh@slac.stanford.edu)
   * [Fabrice Jammes](mailto:fabrice.jammes@in2p3.fr)
```

Comment:
Better then write just after the email: "main mentor (contact me first)"

Reply:
Agree: more explicit